tests/page-objects/forms/pregnancy-visit-form.wdio.page.js

```diff
@@ -0,0 +1,36 @@
+const genericForm = require('./generic-form.wdio.page');
+const dangerSignLabel  =  () =>
```

Comment:
I think there are some extra spaces here. Also applies to next couple of lines. 
```suggestion
const dangerSignLabel = () =>
```